### ISSUE TITLE
[SCREENREADER]: Facility Locator - Investigate map icons to add sr-only label text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#369]
## Goal
The facility locator map icons do not include any text descriptions. They are marked up as divs with `tabIndex="0"` which gives them keyboard focus, but do not announce properly to screen readers. They either say "blank", "div" or do not take [virtual cursor focus](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/) at all. Screenshot attached below.

## Acceptance Criteria
As a screen reader user, I would like three things to happen:
1. The map icons announce the name of the facility or provider when they receive keyboard focus
2. The popup menu would receive keyboard focus when I click or keypress an icon
3. The icon receives focus again when I close the popup

## Definition of Done
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

---
![Screen Shot 2019-06-24 at 3 31 32 PM](https://user-images.githubusercontent.com/934879/60050069-5a40cb00-9695-11e9-8996-2b705c32d33f.png)

**Comment by laflannery** [comment]
Closing this in favor of more recent Facility locator recommendations [#31718](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31718) and https://github.com/department-of-veterans-affairs/va.gov-team/issues/31714
